src/ui/cloud-view.cpp

```diff
@@ -451,6 +459,7 @@ void CloudView::onAccountChanged()
     repos_tab_->refresh();
     starred_files_tab_->refresh();
     activities_tab_->refresh();
+    search_tab_->refresh();
```

Comment:
When account changed, after ` tabs_->removeTab(2, activities_tab_);` we should also remove search tab.